cmd/logcli/main.go

```diff
@@ -19,20 +19,36 @@ import (
 
 var (
 	app        = kingpin.New("logcli", "A command-line for loki.").Version(version.Print("logcli"))
-	quiet      = app.Flag("quiet", "suppress everything but log lines").Default("false").Short('q').Bool()
+	quiet      = app.Flag("quiet", "suppress everything but log entries").Default("false").Short('q').Bool()
 	statistics = app.Flag("stats", "show query statistics").Default("false").Bool()
 	outputMode = app.Flag("output", "specify output mode [default, raw, jsonl]").Default("default").Short('o').Enum("default", "raw", "jsonl")
 	timezone   = app.Flag("timezone", "Specify the timezone to use when formatting output timestamps [Local, UTC]").Default("Local").Short('z').Enum("Local", "UTC")
 
 	queryClient = newQueryClient(app)
 
-	queryCmd   = app.Command("query", "Run a LogQL query.")
+	queryCmd = app.Command("query", `Run a LogQL query.
+
+The default output of this command are log entries (combination of
+timestamp, labels, and log line) along with metainformation about the query
+made to Loki.  The metainformation can be filtered out using the --quiet
```

Comment:
I think I'm still confused what the purpose of the `quiet` flag is all it does is suppress the metainformation about the query?

Reply:
Yeah, I should document it more. Omitting the quiet flag apparently adds the following:

- URL queried 
- "Common labels"
- "Ignoring label key" (if applicable) 

I think that's all? It's passed through all over the place so it's hard to know for sure.

Reply:
maybe it should say "suppress query metadata, use --output=raw to hide labels and timestamps"

Reply:
I'm actually referring to the help text with that last comment, I think the help text should point the users at the --output flag to get rid of labels and timestamps?

Reply:
PTAL, took a shot at these suggestions. I'm a little confused by this too, logcli does a lot of stuff and it hasn't really been documented before.

Reply:
this is better, I like this 👍